source/tv/phantombot/PhantomBot.java

```diff
@@ -1726,13 +1468,6 @@ private static void moveUserConfig() {
         try {
             Files.move(Paths.get("botlogin.txt"), Paths.get("./config/botlogin.txt"));
             Files.move(Paths.get("phantombot.db"), Paths.get("./config/phantombot.db"));
-
-            try {
```

Comment:
Not sure if you moved this, but not deleting these files will cause errors for users that are still on older bot versions.

Reply:
If you move the files, deleting the files on the source location should always fail, shouldn't it? Do I misunderstand something?

Reply:
We saw some behavior where the move() was not fully successful and a delete operation was required.  Keep in mind that we test across several operating systems and flavors of hardware.

Reply:
I reverted it, but I still think, that this code most probably doesn't do what you intend it to do.